perspective_reddit_bot/moderate_subreddit.py

```diff
@@ -199,49 +245,40 @@ def score_subreddit(creds_dict,
   perspective = perspective_client.PerspectiveClient(
     creds_dict['perspective_api_key'])
 
+  current_file_time = datetime.utcnow().strftime('%Y%m%d_%H%M%S')
```

Comment:
`timestamp_to_datetime(datetime.utcnow())` if possible

Reply:
Unfortunately not.